lib/spack/spack/cmd/buildcache.py

```diff
@@ -106,7 +193,9 @@ def find_matching_specs(
     # List of specs that match expressions given via command line
     specs_from_cli = []
     has_errors = False
+    tty.msg('find_matching_specs: going to parse specs')
     specs = spack.cmd.parse_specs(pkgs)
+    tty.msg('find_matching_specs: going to parse specs')
```

Comment:
Appears to be a duplicate of two lines prior. Also was this originally added for debugging purposes? My guess is there should be one call and it should be `tty.debug`